lib/ansible/modules/network/cnos/cnos_vlan.py

```diff
@@ -48,6 +48,8 @@
      cnos.html)
 version_added: "2.3"
 Options:
+    {}
+
     - The following is a table depicting how the overloaded variables are used
```

Comment:
Correct. The script that generates the HTML will only look at that the sections defined in docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html